gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/LocalRegion.java

```diff
@@ -14400,7 +14401,7 @@ protected long calculateDiskIdOverhead(DiskId diskId) {
 
   private AtomicLong memoryBeforeAcquire = new AtomicLong(0L);
 
-  public static long MAX_VALUE_BEFORE_ACQUIRE = 1032 * 100; //100 KB
+  public static long MAX_VALUE_BEFORE_ACQUIRE = 1032 * 10; //10 KB
```

Comment:
In my tests some of the column batch created were smaller than 100KB. And also 100 BK for row tables looks a bit large.